test/workers/test_virtual.py

```diff
@@ -226,3 +228,18 @@ def test_spinup_time(hook):
     dummy = sy.VirtualWorker(hook, id="dummy", data=data)
     end_time = time()
     assert (end_time - start_time) < 0.05
+
+
+@pytest.mark.skip(reason="bug in pytorch version 1.1.0, jit.trace returns raw C function")
```

Comment:
If you're going to skip this function, you also need to mark it with #pragma: no cover